pages/osx/awk.md

```diff
@@ -0,0 +1,24 @@
+# awk
+
+> Find and Replace text within file(s).
+> More information: <https://ss64.com/osx/awk.html>.
+
+- Define the input field separator = regular expression fs:
+
+`awk [ -F fs ]`
+
+- Assign values before prog is executed:
+
+`awk [ -v var=value ]`
```

Comment:
```suggestion
- Assign specific values to variables

`awk {{--assign variable1=value1 --assign variable2=value2 ...}}`
```

Mandatory fix: [ellipsis must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).